app.js

```diff
@@ -36,15 +37,28 @@ function checkParams() {
         } else if (argv.pin && argv.pin.toString().length !== 6) {
             console.error('Pincode must be a 6 digit number \nRefer documentation for more details');
             return;
-        } else if (argv.interval && argv.interval < 5) {
-            console.error('Please provide an interval greater than 5 minutes');
+        } else if (argv.interval && argv.interval < 1) {
```

Comment:
The interval value should be 2 here else the warning message won't match the check.

Reply:
I changed the warning msg also